migration/migrator.py

```diff
@@ -202,9 +211,9 @@ def run_migration(connection, migration, environment, args):
 
 
 def load_migration_module(name, path):
-    spec = importlib.util.spec_from_file_location('module.{}'.format(name), str(path))
-    module = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(module)
+    # TODO: change this to not use depreciated loader.load_module after dropping Python 3.4 support
```

Comment:
deprecated not depreciated

Reply:
I hate how similar these words are to each other and my inability for my brain to handle it.